src/index.js

```diff
@@ -1,20 +1,16 @@
 import readlineSync from 'readline-sync';
-import getRandomNumber from './randomCount.js';
 
 const numberOfRound = 3;
-const brainEven = () => {
+const theCoreOfGames = (brainRulls) => {
   console.log('Welcome to the Brain Games!');
   const userName = readlineSync.question('May I have your name? ');
   console.log(`Hello, ${userName}!`);
-
-  console.log('Answer "yes" if the number is even, otherwise answer "no".');
+  const [descriptionOfGame] = brainRulls();
+  console.log(descriptionOfGame);
```

Comment:
Plz separate the logical parts of the code with empty lines. Now the code is hard to read

Reply:
ok, added, I hope correct)